dataforest/core/PlotMethods.py

```diff
@@ -28,14 +29,29 @@ def plot_method_lookup(self):
     def plot_methods(self):
         return self.__class__.PLOT_METHODS
 
+    @property
+    def plot_kwargs_defaults(self):
+        return self.__class__.PLOT_KWARGS_DEFAULTS
+
+    @property
+    def plot_kwargs(self):
+        return self.__class__.PLOT_KWARGS
+
     def _wrap(self, method):
         """Wrap with mkdirs and logging"""
 
         @wraps(method)
         def wrapped(branch, method_name, *args, stop_on_error: bool = False, **kwargs):
             try:
                 process_run = branch[branch.current_process]
-                plot_dir = process_run.plot_map[method_name].parent
+                plot_name = get_plot_name_from_plot_method(
```

Comment:
Could probably be abstracted away to a `_get_plot_dir` method to avoid crowding the wrapper and having a lot of raw lines in the try block. Abstracting it away would also clarify what these lines are doing by virtue of the method name

Reply:
Seems like you resolved this!